### PR TITLE
Update documentation about System32 directory in FIM

### LINKED ISSUE #3586: Request to update our documentation about FIM configuration
Lets update our documentation about [FIM configuration](https://documentation.wazuh.com/current/user-manual/capabilities/file-integrity/fim-configuration.html#configuring-who-data-monitoring) 

To configure properly File Integrity Monitoring for directories under 'System32' in a 64 bit Windows machine we need to add a note, or perhaps expand the existing one (as it was done for the env. variable 'C:\Program Files (x86)\Common Files'):

image

We need to inform our customers the following:

To be able to scan any directory under 'System32', including it, in a 64 bit system, replace %windir%\System32 for %windir%\Sysnative

As an example, if you want to scan IIS' 'inetsrv\config' configuration directory for changes, you need to use:

	<directories check_all="yes" whodata="yes" report_changes="yes">%WINDIR%\Sysnative\inetsrv\config</directories>

There is more information about this directive from Microsoft at this link: [File System Redirector](https://docs.microsoft.com/en-us/windows/win32/winprog64/file-system-redirector)

### PR DESCRIPTION
**Pull request by GabrielEValenzuela**
## Description

This PR closes #3586. It updates how to monitor Windows system locations in FIM according to the latest changes in the core product (at https://github.com/wazuh/wazuh/issues/14796).

## Checks
- [x] Compiles without warnings.
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.
- [ ] Uses three spaces indentation.
- [ ] Adds or updates meta descriptions accordingly.
- [ ] Updates the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/user-manual/capabilities/file-integrity/fim-configuration.rst [hunk 1]
```diff
@@ -180,6 +180,44 @@ To configure the Windows registries, it is necessary to create a list of those r
   </syscheck>
 
 
+.. _how_to_fim_windows_directories:
+
+Configuring Windows directories
+-------------------------------
+
+To configure directories within the Windows directory (Also known as ``%WINDIR%``), it is necessary to take into account that in 64-bit architecture systems, there are two special folders: ``System32`` which is reserved for those 64-bit DLLs, and ``SysWOW64`` for all 32-bit DLLs.
```
Comment (jotacarma90) [inline source/user-manual/capabilities/file-integrity/fim-configuration.rst:188]:
```suggestion
To configure Windows special directory ``%WINDIR%\system32``, it is necessary to take into account that in 64-bit architecture systems: 
1) **System32** is reserved for those 64-bit DLLs.
2) **SysWOW64** for all 32-bit DLLs.

```

File: source/user-manual/capabilities/file-integrity/fim-configuration.rst [hunk 2]
```diff
@@ -180,6 +180,44 @@ To configure the Windows registries, it is necessary to create a list of those r
...
+The 32-bit processes running in a 64-bit environment, when they want to access Sytem32, they do it through a virtual folder called ``Sysnative``.
+
+From version 4.4.0 of Wazuh, this redirection has been disabled and it is possible to access directly to ``System32``, so those configurations that have as monitoring directory ``%WINDIR%/Sysnative`` from now on will generate alerts, synchronizations with the database and other functions with ``%WINDIR%/System32``. That means, ``%WINDIR%/Sysnative`` or ``%WINDIR%/System32`` represent the same directory and both path are valids.
```
Comment (jotacarma90) [inline source/user-manual/capabilities/file-integrity/fim-configuration.rst:191]:
```suggestion
From version 4.4.0 of Wazuh, this redirection has been disabled and it is possible to access directly to ``System32``, so monitoring directories ``%WINDIR%/System32`` or ``%WINDIR%/Sysnative`` will be equivalent, and the path shown in the alerts will be ``%WINDIR%/System32``.
```

**Review by jotacarma90 — APPROVED** [review]
LGTM

**Review by chemamartinez — CHANGES_REQUESTED** [review]

File: source/user-manual/capabilities/file-integrity/fim-configuration.rst [hunk 3]
```diff
@@ -180,6 +180,47 @@ To configure the Windows registries, it is necessary to create a list of those r
...
+Configuring Windows directories
+-------------------------------
+
+To configure Windows special directory ``%WINDIR%\system32``, it is necessary to take into account that in 64-bit architecture systems: 
+1) **System32** is reserved for those 64-bit DLLs.
+2) **SysWOW64** for all 32-bit DLLs.
+
+The 32-bit processes running in a 64-bit environment, when they want to access Sytem32, they do it through a virtual folder called ``Sysnative``.
+
+From version 4.4.0 of Wazuh, this redirection has been disabled and it is possible to access directly to ``System32``, so monitoring directories ``%WINDIR%/System32`` or ``%WINDIR%/Sysnative`` will be equivalent, and the path shown in the alerts will be ``%WINDIR%/System32``.
```
[the remark this answers is not in the document]
Comment (GabrielEValenzuela) [reply source/user-manual/capabilities/file-integrity/fim-configuration.rst]: Done!

**Review by chemamartinez — APPROVED** [review]

File: source/user-manual/capabilities/file-integrity/fim-configuration.rst [hunk 4]
```diff
@@ -179,6 +179,46 @@ To configure the Windows registries, it is necessary to create a list of those r
...
+-------------------------------
+
+In 64-bit architecture systems, you can locate 32-bit and 64-bit DLLs in a special way.
+
+-  ``System32`` is reserved for 64-bit DLLs.
+-  ``SysWOW64`` is reserved for all 32-bit DLLs.
+
+Furthermore, 32-bit processes running in 64-bit environments access ``Sytem32`` through a virtual folder called ``Sysnative``.
+
+But this redirection has been disabled and you can access ``System32`` directly. Monitoring ``%WINDIR%/System32`` and ``%WINDIR%/Sysnative`` directories is equivalent and Wazuh shows the path ``%WINDIR%/System32`` in the alerts. ``SysWOW64`` is a different directory. To monitor ``%WINDIR%/SysWOW64``, you must add it to the ``ossec.conf`` configuration file.
```
Comment (javimed) [inline source/user-manual/capabilities/file-integrity/fim-configuration.rst:192]:
```suggestion
But we disabled this redirection and you can access ``System32`` directly. Monitoring ``%WINDIR%/System32`` and ``%WINDIR%/Sysnative`` directories is equivalent and Wazuh shows the path ``%WINDIR%/System32`` in the alerts. ``SysWOW64`` is a different directory. To monitor ``%WINDIR%/SysWOW64``, you must add it to the ``ossec.conf`` configuration file.
```

**Review by javimed — APPROVED** [review]

**Pull request merged**